### PR TITLE
MessageBus: Throw error if Kafka Broker is down

### PR DESCRIPTION
**Pull request by selvakumaar5496**
Signed-off-by: Selvakumar <selva.k.nambi@seagate.com>

### CODE REVIEW

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Address the comment

File: py-utils/src/utils/message_bus/message_broker_collection.py [hunk 1]
```diff
@@ -124,6 +129,11 @@ def _get_metadata(self, admin: object):
             raise MessageBusError(errno.EINVAL, "Unable to list message type. \
                 %s", e)
 
+    @staticmethod
+    def _error_cb(err):
+        if err.code() == KafkaError._ALL_BROKERS_DOWN:
+            raise MessageBusError(err.code(), "All Brokers are down. %s", err)
```
[the remark this answers is not in the document]
Comment (selvakumaar5496) [reply py-utils/src/utils/message_bus/message_broker_collection.py]: Done

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good.

**Pull request merged**